core/src/test/java/org/apache/iceberg/rest/requests/TestCreateNamespaceRequest.java

```diff
@@ -166,6 +139,6 @@ public void assertEquals(CreateNamespaceRequest actual, CreateNamespaceRequest e
 
   @Override
   public CreateNamespaceRequest deserialize(String json) throws JsonProcessingException {
-    return mapper().readValue(json, CreateNamespaceRequest.class).validate();
```

Comment:
note that we don't have to explicitly validate anymore as objects created via Immutables are by definition always in a valid state.